src/06-Driver/15-Disposing-Components.md

```diff
@@ -9,5 +9,4 @@ To compile the next file and view its results in the browser, use these instruct
 ```bash
 spago bundle-app -m Driver.DisposingComponents -t assets/driver/disposing-components.js
 parcel serve assets/driver/disposing-components.html -o disposing-components--parcelified.html --open
-# After the page opens, check your browser's console to see the messages
```

Comment:
This comment was false - that example has no messages in console.

Reply:
Yup! Thanks